### ISSUE TITLE
2.6.0 cannot generate a new project

### ISSUE THREAD

**Issue by saturnflyer** [strongqa/howitzer#334]
## Meta
Howitzer Version: 
2.6.0

Driver and browser information:
🤷 

Gem Versions:
🤷 

## Expected Behavior

`howitzer new project --rspec` should generate a new project.

## Actual Behavior

```
error: No such file or directory @ rb_sysopen - /Users/------/.asdf/installs/ruby/3.2.0/lib/ruby/gems/3.2.0/gems/howitzer-2.6.0/generators/root/templates/.rubocop.yml.erb
```

## Steps to reproduce

1. gem install howitzer
2. howitzer new project --rspec

**Comment by romikoops** [comment]
Hello @ saturnflyer
Thank you for posting this bug, it is really pitty we missed that. I will fix and release a new version ASAP

**Comment by romikoops** [comment]
fixed in 2.6.1
